lib_runtime/mirage_runtime.mli

```diff
@@ -77,6 +77,17 @@ val delay : int Term.t
     Defaults to 0. Useful for tenders and environments that take some time to
     bring devices up. *)
 
+(** {2 Name} *)
+
+val set_name : string -> unit
+(** Set the default name. *)
```

Comment:
Should this be behind `(**/**)`? I don't mind either way, but if not we could change the doc for `val name` to say "The *current* name of the unikernel". It would be nice to know if one can safely assume the name will not change during the lifetime of the unikernel.